connectors/jdbc/pom.xml

```diff
@@ -50,18 +50,25 @@
       <version>12.6.1.jre11</version>
     </dependency>
 
-    <dependency>
-      <groupId>com.mysql</groupId>
-      <artifactId>mysql-connector-j</artifactId>
-      <version>8.4.0</version>
-    </dependency>
-
     <dependency>
       <groupId>org.postgresql</groupId>
       <artifactId>postgresql</artifactId>
       <version>42.7.3</version>
     </dependency>
 
+    <!-- Mariadb is used because of an issue with the MySQL ConnectorJ licence (GPL). We're not mentioning Mariadb in our docs -->
+    <dependency>
+      <groupId>org.mariadb.jdbc</groupId>
+      <artifactId>mariadb-java-client</artifactId>
+      <version>3.3.3</version>
+    </dependency>
+
+    <dependency>
+      <groupId>com.mysql</groupId>
+      <artifactId>mysql-connector-j</artifactId>
```

Comment:
Double-checking: is this test dependency still here intentionally?

If so, I'm curious why we still need it.

This is a non-blocking comment 😄

Reply:
scope is `test` so I assume it won't be exported in our artifact.
I left a comment in the PR description you probably missed it :'(

Reply:
Indeed I missed it in my rush 😢 

Thanks!